webapp/src/js/controllers/messages-content.js

```diff
@@ -115,9 +115,35 @@ angular.module('inboxControllers').controller('MessagesContentCtrl',
         getContactable(id, type),
         MessageContacts.conversation(id)
       ])
+        .catch(function(err) {
+          if (err.code === 404) {
+            return $q.all([Promise.resolve(null), MessageContacts.conversation(id)]);
+          }
+
+          throw err;
+        })
         .then(function(results) {
           var contactModel = results[0];
           var conversation = results[1];
+          if (!contactModel) {
+            const firstTaskWithContact = _.find(conversation[0].doc.tasks, 
```

Comment:
You do not need to use underscore as a shim for find, our [minimum chrome is 53](https://github.com/medic/medic-docs/blob/master/installation/supported-software.md#dependencies) and [support was added in 45](https://caniuse.com/#feat=array-find).